examples/soap-calculator/src/index.ts

```diff
@@ -0,0 +1,14 @@
+import {SoapCalculatorApplication} from './application';
+import {ApplicationConfig} from '@loopback/core';
+import {RestServer} from '@loopback/rest';
```

Comment:
Since we can remove lines 10-11 below because of #1572, this import can also be removed.